examples/wearable/UIComponents/contents/user-inputs-elements/pickers/two-items-picker.js

```diff
@@ -0,0 +1,114 @@
+(function (window) {
+	"use strict";
+
+	var page = document.getElementById("two-items-spin-page"),
+		descriptions = document.getElementById("descriptions"),
+		content = page.querySelector(".ui-content"),
+		element = page.querySelector(".ui-spin"),
+		spin = null,
+		stage = 0;
+
+	function onClickChange() {
+		switch (stage) {
+			case 0 :
+				descriptions.innerHTML = ".value(9)";
```

Comment:
what is .value(9) ?

Reply:
Fixed. This is description for action from test scenario.